packages/react-components/src/localization/LocalizationProvider.tsx

```diff
@@ -84,6 +86,9 @@ export interface ComponentStrings {
   /* @conditional-compile-remove(one-to-n-calling) */ /* @condtional-compile-remove(one-to-n-calling) */
   /** Strings for HoldButton */
   holdButton: HoldButtonStrings;
+  /* @conditional-compile-remove(call-readiness) */ /* @conditional-compile-remove(unsupported-browser) */
```

Comment:
Before when two features were above a conditional compile they implied that id _either_ feature1 _or_ feature2 was stabilized we want to enabled this feature. But in this case we only want unsupported-browser to enable this feature so remove the `/* @conditional-compile-remove(call-readiness) */` flag if possible